Update export-or-backup-using-dumpling.md (#3454)
pingcap/docs: export-or-backup-using-dumpling.md

For backups of SST files (KV pairs) or backups of incremental data that are not sensitive to latency, refer to [BR](/br/backup-and-restore-tool.md). For real-time backups of incremental data, refer to [TiCDC](/ticdc/ticdc-overview.md).

> For detailed usage of Dumpling, use the `--help` command to view it.

Reviewer: Why not include the English user guide (https://github.com/pingcap/dumpling/blob/master/docs/en/user-guide.md)?

Author: Nice catch. Updated.

> For detailed usage of Dumpling, use the `--help` command to view it.

Author: ```suggestion
For detailed usage of Dumpling, use the `--help` command or refer to [Dumpling User Guide](https://github.com/pingcap/dumpling/blob/master/docs/en/user-guide.md).
```

### Filter the exported data

#### Use the `--where` command to filter data

By default, Dumpling exports the tables of the entire database except the tables in the system databases. You can use `--where <SQL where expression>` to select the records to be exported.

{{< copyable "shell-regular" >}}

```shell
./dumpling \
  -u root \
  -P 4000 \
  -h 127.0.0.1 \
  -o /tmp/test \
  --where "id < 100"
```

The above command exports the data that matches `id < 100` from each table.

#### Use the `--filter` command to filter data

Dumpling can filter specific databases or tables by specifying the table filter with the `--filter` command. The syntax of table filters is similar to that of `.gitignore`. For details, see [Table Filter](/table-filter.md).

{{< copyable "shell-regular" >}}

```shell
./dumpling \
  -u root \
  -P 4000 \
  -h 127.0.0.1 \
  -o /tmp/test \
  --filter "employees.*"
  --filter "*.WorkOrder"
```

The above command exports all the tables in the `employees` database and the `WorkOrder` tables in all databases.

#### Use the `-B` or `-T` command to filter data

> Dumpling can also export specific databases or tables with the `-B` or `-T` command.

Reviewer: ```suggestion
Dumpling can also export specific databases with the `-B` command or specific tables with the `-T` command.
```
